Adjusted Inventory and Group API hostFilter documention (#3128)
cfengine/documentation: api/enterprise-api-ref/personal-groups.markdown

    * **hostContextExclude** *(array)*  Optional parameter.
      Excludes results that concern hosts which have specified CFEngine context (class) set. Hosts that have at least one
      of the specified contexts set will be excluded from the results.
    * **hostFilter** *(json object)* Optional parameter.
        * **includes** *(json object)* Optional parameter.
          Object that specifies hosts to be included.
            * **includeAdditionally** *(boolean)* Default: `false`

>               Defines if hosts will be included additionally on top of the inventory filters or class filters.

Reviewer: I don't immediately understand from this what happens in both true and false cases.

So includes with includeAdditionally=false won't include the hosts given by the current filter? What is the point of this flag?

Does false mean that the hosts found with the current filter will only be included if already part of inventory or class filters?

And so true means that the hosts found with the current filter will be included regardless of the results given by the inventory or class filters?

Reviewer: maybe "on top" would be better said as 
```suggestion
              Defines if hosts will be added to the results returned by inventory filters or class filters.
```

        "operator":"value2"
      }
    },

>     "hostFilter": {

Reviewer: Is this fairly large example repeated a few times? Maybe we could just do it once up-front and then refer to it later.

Also, showing some sample results that take advantage of each item in the example request would be nice.
